README.md

```diff
@@ -11,7 +11,7 @@
 
 ### iOS
 
-* iOS 7+ 
+* iOS 7+
```

Comment:
This can be avoided in the future with a `.editorconfig` file like so:

https://github.com/jhabdas/react-native-webpack-starter-kit/blob/master/.editorconfig#L15-L16